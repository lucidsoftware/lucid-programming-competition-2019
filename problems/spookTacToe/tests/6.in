9 9
xx#......
x.#x.....
..#.#....
xx.#.....
.#......#
....##...
.x.x....#
.xx#.o..x
x.#......