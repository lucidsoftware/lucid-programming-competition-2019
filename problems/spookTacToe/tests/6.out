xx#oxoxox
xo#xoxoxo
ox#x#xoxo
xxo#xoxox
x#ooxoxo#
ooxx##oxo
xxoxoxox#
oxx#ooxox
xo#oxoxox